README.md

```diff
@@ -343,16 +343,18 @@ extra_docker_arguments: []  # Additional arguments for `docker container create`
 
 The [toastfile](https://github.com/stepchowfun/toast/blob/main/toast.yml) for Toast itself is a comprehensive real-world example.
 
-## Cache configuration
+## Configuration
 
-Toast supports local and remote caching. By default, only local caching is enabled. Remote caching requires that the Docker Engine is logged into a Docker registry (e.g., via `docker login`).
-
-The caching behavior can be customized with a configuration file. The default location of the configuration file depends on the operating system:
+The behavior can be customized with a configuration file. The default location of the configuration file depends on the operating system:
 
 - For macOS, the default location is `$HOME/Library/Application Support/toast/toast.yml`.
 - For other Unix platforms, Toast follows the [XDG Base Directory Specification](https://specifications.freedesktop.org/basedir-spec/basedir-spec-latest.html). The default location is `$XDG_CONFIG_HOME/toast/toast.yml` or `$HOME/.config/toast/toast.yml` if `XDG_CONFIG_HOME` is not set to an absolute path.
 - For Windows, the default location is `{FOLDERID_RoamingAppData}\toast\toast.yml`.
 
+### Cache configuration
+
+Toast supports local and remote caching. By default, only local caching is enabled. Remote caching requires that the Docker Engine is logged into a Docker registry (e.g., via `docker login`).
+
 The configuration file has the following schema and defaults:
```

Comment:
This doesn't seem to be accurate any longer after this PR.

Reply:
Perhaps just adding a blurb, "Only when using the default `docker` CLI..."?

Reply:
Do these existing config fields not work with Podman?

Reply:
I would expect this line to say something like:

> The caching behavior can be configured with the following options:

The reason the line is incorrect as of now is because the this isn't the complete schema, since there is also the new `docker_cli` key described in a different subsection. My apologies if it wasn't clear what my feedback was.

Reply:
> Do these existing config fields not work with Podman?

They do. My thinking on splitting them apart was the `docker_cli` option doesn't really have much to do with caching. To me it was, "These four options are for caching, and now there is another option in the config in it's own category."

Reply:
Got it. I'm fine with splitting up the documentation like this into two sections. I just think this sentence needs to change, because currently it's written as if this is the schema for the whole config file. The sentence says "The configuration file has the following schema", but that's a lie now; the following 5 fields are only _part_ of the schema now.

Do you understand what I am saying? It's just a minor point about a single line.